backends/LxcBackend.py

```diff
@@ -42,10 +44,12 @@ def provision(self, isMaster=False, isRenet=False):
                 print("Container seems to have failed to start (no IP)")
                 sys.exit(1)
 
-        #filesdir = os.path.dirname(os.path.realpath(sys.modules['__main__'].__file__)) + "/" + path + scriptname
-        #if os.path.isfile(filesdir):
-            ret = c.attach_wait(lxc.attach_run_command, ["env"] + ["MILXCGUARD=TRUE", "HOSTLANG="+os.getenv("LANG")] + [
-                            getInterpreter(filesdir), "/mnt/lxc/" + path + scriptname], env_policy=lxc.LXC_ATTACH_CLEAR_ENV)
+        # filesdir = os.path.dirname(os.path.realpath(sys.modules['__main__'].__file__)) + "/" + path + scriptname
+        # if os.path.isfile(filesdir):
+            ret = c.attach_wait(lxc.attach_run_command, ["env"]
```

Comment:
perhaps ["env"] should be starting the second line and we could better indent the different "subparameters" levels ?

Reply:
The problem with doing this:
```python
ret = c.attach_wait(lxc.attach_run_command,
                ["env"] + args + [getInterpreter(filesdir), "/mnt/lxc/" + path + scriptname], env_policy=lxc.LXC_ATTACH_CLEAR_ENV)
```
is that you get the following error: `continuation line under-indented for visual indent (E128)`
So it seems better to me to stay as such.

Reply:
I thought something like that :


```python
            ret = c.attach_wait(lxc.attach_run_command,
                                ["env"] + ["MILXCGUARD=TRUE", "HOSTLANG=" + os.getenv("LANG")]
                                + [getInterpreter(filesdir), "/mnt/lxc/" + path + scriptname],
                                env_policy=lxc.LXC_ATTACH_CLEAR_ENV)
```

There are still E501 (line too long), but I think that ["env"], as the beginning of the second argument to attach_wait, should start a line, aligned with lxc.attach_run_command (and in that case, each beginning of argument to attach_wait should start a line). No ?

(with this precise alignment, I don't have E128)

Reply:
Ok, I wll fix this.